### ISSUE TITLE
RPC unavailable on Local Diag --> Unable to Reboot nor to Purge Data

### ISSUE THREAD

**Issue by fouad-semaan** [NebraLtd/helium-miner-software#311]
**Describe the bug**
I have been monitoring a couple of miners for many days:

Behaviour: Switching status between 100% Sync and RPC unavailable on Local Diag continuously. 

Rebooting: Unable to reboot.
Purge Data: Unable to Purge Data (see error Loop below)

```
Killing service 'upnp sha256:059223cdbf528c09361826a200e6baa8d4234ad6a9076c5a088d25f9255d53ff'
Failed to kill service 'upnp sha256:059223cdbf528c09361826a200e6baa8d4234ad6a9076c5a088d25f9255d53ff' due to '(HTTP code 409) unexpected - You cannot remove a running container 8af61954afd2aed9fe7e7215db68ff0c3465249ebff625210226dbd84f4c78ae. Stop the container before attempting removal or force remove '

Killing service 'upnp sha256:059223cdbf528c09361826a200e6baa8d4234ad6a9076c5a088d25f9255d53ff'
Failed to kill service 'upnp sha256:059223cdbf528c09361826a200e6baa8d4234ad6a9076c5a088d25f9255d53ff' due to '(HTTP code 409) unexpected - You cannot remove a running container 8af61954afd2aed9fe7e7215db68ff0c3465249ebff625210226dbd84f4c78ae. Stop the container before attempting removal or force remove '

Killing service 'upnp sha256:059223cdbf528c09361826a200e6baa8d4234ad6a9076c5a088d25f9255d53ff'
Failed to kill service 'upnp sha256:059223cdbf528c09361826a200e6baa8d4234ad6a9076c5a088d25f9255d53ff' due to '(HTTP code 409) unexpected - You cannot remove a running container 8af61954afd2aed9fe7e7215db68ff0c3465249ebff625210226dbd84f4c78ae. Stop the container before attempting removal or force remove '
```

e.g.:
https://dashboard.balena-cloud.com/devices/9a99fc079d4be877ab23278c86b3247f/summary

**Comment by shawaj** [comment]
@fouad-semaan you can fix this using:

```
balena-engine container ls
```

Then get the container ID for the container with the issue... Then you can run...

```
balena-engine container rm -f 97db54d01814
```

Where `97db54d01814` is the container ID you copied from above.

This will force kill the container

**Comment by shawaj** [comment]
IIRC @KevinWassermann94 already documented this 409 issue somewhere with the fix?

**Comment by shawaj** [comment]
This is possibly a duplicate of #266

**Comment by fouad-semaan** [comment]
Yes this is partially duplicated regarding error 409 and Kevin has already executed the commands above on that miner. 

The remaining issue is the RPC unavailable. It still showing up and each time in different aspect.

e.g. the storage of this one has surpassed 24GB so I decided to purge it. After the purge, it had the RPC until I rebooted it.

https://dashboard.balena-cloud.com/devices/6d746d247172b7c7f47603213e390c29/summary
